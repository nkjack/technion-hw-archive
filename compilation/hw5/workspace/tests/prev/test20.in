void maxTestsStressArithmeticOperations() {
	int x = 10 / 1 / 1 / 1 / 1 / 1 / 1 * 1 * 1 * 1 * 1 * 1 / 1 * 1 / 1 + 1 / 1 * 1 / 1;
	printi(x);
}

void main() {
     maxTestsStressArithmeticOperations();
     print("\nI hate this course\n");
}
