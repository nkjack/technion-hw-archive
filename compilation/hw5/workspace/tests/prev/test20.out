Loaded: ./exceptions.s
11
I hate this course
